### PR TITLE
Fix wrong chinese translation (SELinux note)

### PR DESCRIPTION
**Pull request by life1347**
The original sentence is
```
You can leave SELinux enabled if you know how to configure it but it may require settings that are not supported by kubeadm.
```
while the old chinese translation means to keep it disabled.

### CODE REVIEW

**Comment by life1347** [comment]
I signed it

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -402,8 +402,7 @@ systemctl enable --now kubelet
 
   你必须这么做，直到 kubelet 做出对 SELinux 的支持进行升级为止。
 
-- 你可以保持 SELinux 处于弃用状态，前提是你知道如何配置它，不过这也意味着有些
-  配置是 kubeadm 所不支持的。
+- 如果你知道如何配置 SELinux 则可以将其保持启用状态，但可能需要设定部分 kubeadm 不支持的配置。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:405]:
```suggestion
- 如果你知道如何配置 SELinux 则可以将其保持启用状态，但可能需要设定 kubeadm 不支持的部分配置。
```

**Comment by tengqm** [comment]
Note that the upstream English version has some more changes that need to be synced. Please do sync them.

**Comment by life1347** [comment]
@tengqm Thanks for your comment. I've updated the page with the upstream English version.

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -527,7 +523,7 @@ Please mind, that you **only** have to do that if the cgroup driver of your CRI
 is not `cgroupfs`, because that is the default value in the kubelet already.
 -->
 进一步的相关细节，可参阅
-[使用配置文件来执行 kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)。
+[使用配置文件来执行 kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) 以及 [KubeletConfiguration](/docs/reference/config-api/kubelet-config.v1beta1/)
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: pls sync line 520 above for future change tracking.

Comment (life1347) [reply content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: done. thanks

**Pull request merged**